bluepyopt/ephys/morphologies.py

```diff
@@ -50,17 +50,22 @@ def __init__(
             do_replace_axon=False,
             do_set_nseg=True,
             comment='',
-            replace_axon_hoc=None):
+            replace_axon_hoc=None,
+            morph_modifiers=None):
```

Comment:
i think we're going to have a check in create_hoc of the model to make sure morph_modifiers is None. Because if it's not none, the hoc code conversion will not be valid.

Reply:
I'm on that, I will push in a minute.

Reply:
So I basically allowed the user to add custom hoc files in a list, which are then appended to the replace_axon hoc file. I hope it'll work!

Reply:
changing that name may also break other codes, that's why I left it as it is

Reply:
I thought the best was to add another argument to provide a nseg_frequency, which will be =40 by default